src/main/java/seedu/address/logic/parser/EditGroupCommandParser.java

```diff
@@ -37,8 +37,12 @@ public EditGroupCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
                     EditGroupCommand.MESSAGE_USAGE), ive);
         }
+        argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_NAME, PREFIX_TAG);
         String newGroupName = argMultimap.getValue(PREFIX_NAME).orElse("");
-        Collection<Tag> tags = ParserUtil.parseTagsForEdit(argMultimap.getAllValues(PREFIX_TAG)).orElse(null);
+        Collection<Tag> tags = null;
+        if (argMultimap.getValue(PREFIX_TAG).isPresent()) {
+            tags = ParserUtil.parseTagsForEdit(argMultimap.getAllValues(PREFIX_TAG)).get();
+        }
```

Comment:
We need to set tags to an empty collection if it doesn't exist in the edit command. I'll fix this later